grafana-plugin/integration-tests/utils/userSettings.ts

```diff
@@ -2,19 +2,23 @@ import { Locator, Page } from '@playwright/test';
 
 import { clickButton, fillInInputByPlaceholderValue, selectDropdownValue } from './forms';
 import { closeModal } from './modals';
-import { goToOnCallPage } from './navigation';
+import { OnCallPage, goToOnCallPage } from './navigation';
 import { getPhoneNumber, getVerificationCodeFromSms, waitForSms } from './phone';
 
 type NotifyBy = 'SMS' | 'Phone call';
 
 const openUserSettingsModal = async (page: Page): Promise<void> => {
-  await goToOnCallPage(page, 'users');
+  await goToOnCallPage(page, OnCallPage.USERS);
   await clickButton({ page, buttonText: 'View my profile' });
   await page.locator('text=To edit user details such as Username, email, and role').waitFor({ state: 'visible' });
 };
 
 const getForgetPhoneNumberButton = (page: Page): Locator => page.locator('button >> text=Forget Phone Number');
 
+export const openViewMyProfile = async (page: Page): Promise<void> => {
+  await openUserSettingsModal(page);
+};
```

Comment:
do we need this function? Could we just call `openUserSettingsModal(page)` directly instead?

Reply:
The plan was to add more stuff to it, todo for next time. I changed it as you suggested in the meanwhile.